rector.php

```diff
@@ -132,6 +135,11 @@
     $rectorConfig->rule(MakeInheritedMethodVisibilitySameAsParentRector::class);
     $rectorConfig->rule(SimplifyEmptyArrayCheckRector::class);
     $rectorConfig->rule(NormalizeNamespaceByPSR4ComposerAutoloadRector::class);
+    $rectorConfig
+        ->ruleWithConfiguration(TypedPropertyFromAssignsRector::class, [
+            // Set to false if you use in libraries, or it does create breaking changes.
+            TypedPropertyFromAssignsRector::INLINE_PUBLIC => false,
```

Comment:
It false by default, no need to define in config for false value

Reply:
It comes from a template file in devkit.

Reply:
For devkit, that can be on purpose so user know when to enable it, I think it can be removed in here

Reply:
I followed the instruction in the devkit template.

If we need to remove the default value, please update the devkit comment.
Otherwise, the code will revert again when I or someone else copy the devkit template to update in the future.

Reply:
Ok, I created PR https://github.com/codeigniter4/devkit/pull/61 for it.

Reply:
Removed the config line.